vscode/src/extension.ts

```diff
@@ -142,3 +149,16 @@ async function loadLanguageService(baseUri: vscode.Uri) {
   await loadWasmModule(wasmBytes);
   return await getLanguageService();
 }
+
+export class QsTextDocumentContentProvider
+  implements vscode.TextDocumentContentProvider
+{
+  onDidChange?: vscode.Event<vscode.Uri> | undefined;
+  provideTextDocumentContent(
+    uri: vscode.Uri,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    token: vscode.CancellationToken
+  ): vscode.ProviderResult<string> {
+    return provideTextDocumentContent(uri.path);
```

Comment:
Do we only store docs internally via the path, not the full Uri? This could be an issue later if we support referencing docs via other protocols (e.g. I have in my project "src/foo.qs" and "http://somesite.com/src/foo.qs" then do they have the same path). Is it possible for the path part of the std lib files to conflict with user files?

Reply:
This provider is only used when resolving core/std library files as a fallback for having missing source. Any local files should load just fine.